api/all/src/main/java/io/opentelemetry/api/logs/Logger.java

```diff
@@ -33,4 +33,17 @@ public interface Logger {
    * LogRecordBuilder#emit()}.
    */
   LogRecordBuilder logRecordBuilder();
+
+  /**
+   * Return a {@link LogRecordBuilder} to emit an event.
+   *
+   * @param eventName the event name, which identifies the class or type of event. Event with the
+   *     same name are structurally similar to one another. Event names are subject to the same
+   *     naming rules as attribute names. Notably, they are namespaced to avoid collisions. See <a
+   *     href="https://opentelemetry.io/docs/specs/semconv/general/events/">event.name semantic
+   *     conventions</a> for more details.
+   */
+  default EventBuilder eventBuilder(String eventName) {
+    return DefaultLogger.getInstance().eventBuilder(eventName);
```

Comment:
nit: I wish this was encapsulated into `DefaultLogger.eventBuilder(eventName)`...

Reply:
sorry, I didn't follow what you mean here?

Reply:
oh, just literally that I wish there was a static method on `DefaultLogger` called `eventBuilder(name)` that would prevent us from having to go thru the instance here.